catalyst/dl/callbacks/metrics/cmc.py

```diff
@@ -197,6 +125,14 @@ def on_loader_start(self, runner: "IRunner"):
 
     def on_loader_end(self, runner: "IRunner"):
```

Comment:
btw, let's also add possibility to pass `query_size` and `gallery_size` during Callback init, rather than hardcode MLDataset usage.
cc @elephantmipt

Reply:
rejected